src/backend/distributed/planner/multi_explain.c

```diff
@@ -182,6 +182,8 @@ MultiExplainOneQuery(Query *query, IntoClause *into, ExplainState *es,
 		return;
 	}
 
+	ExplainOpenGroup("Distributed Query", NULL, true, es);
```

Comment:
Well, I want to understand the usage pattern of `labeled` parameter of `ExplainOpenGroup`. Why do we set it to `true` in some cases and `false` in others? 
For instance, why in the case of `Tasks` it's false and in the case of`Task` it's `true`?